sound/mocp/Makefile

```diff
@@ -43,16 +43,18 @@ TARGET_CFLAGS+=-D_GNU_SOURCE
 TARGET_CPPFLAGS+=-P
 
 CONFIGURE_ARGS+= \
-		$(if $(CONFIG_BUILD_PATENTED),,--without-mp3) \
 		--enable-shared \
 		--disable-static \
 		--disable-debug \
 		--without-speex \
-		--without-samplerate \
-		--without-curl \
+		--without-aac \
 		--without-flac \
+		--without-mp3 \
```

Comment:
mp3 refers to libmad which is not needed as ffmpeg handles mp3 just fine and is already linked.